script.js

```diff
@@ -13,14 +13,35 @@ function closeNav() {
 const submit = document.getElementById('submit');
 submit.addEventListener('click', (e) => {
   const email = document.getElementById('email').value;
+  const name1 = document.getElementById('name').value;
+  const message = document.getElementById('message').value;
   const emailLower = email.toLowerCase();
   const text = document.getElementById('text');
+  const userInputData = {
+    name1,
+    email,
+    message,
+  };
+  const savedData = JSON.stringify(userInputData);
```

Comment:
- [ ] Kindly fix the linters errors instead of skipping them, this will make your code follow a general guideline provided by microverse for all students. You can refer to lines 1 to 4 of the script.js

```
/* eslint-disable linebreak-style */
/* eslint-disable no-plusplus */
/* eslint-disable no-unused-vars */
/* eslint-disable linebreak-style */
```

Reply:
@Ismailco there is no fixing for this as it is general problem and my fellow students have done the same and got approved.